ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.test.tsx

```diff
@@ -129,51 +139,52 @@ describe('TimeSeriesChartPanel', () => {
     renderPanel();
 
```

Comment:
Throughout this test, these legend list item lookups need to be done at the time they're being checked (instead of once and reused) because they are virtualized and the underlying element can change.